### PR TITLE
Update PPDataCollector doc string

### PR DESCRIPTION
**Pull request by sestevens**
### Summary of changes

- We got some feedback about a misleading doc string in `PPDataCollector`. The device data collected can be used for cards, not just PayPal and Venmo transactions, so this PR updates that comment.

### Checklist

- ~Added a changelog entry~

- @sestevens

### CODE REVIEW

File: Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift [hunk 1]
```diff
@@ -17,10 +20,9 @@ import Security
     }
 
     /**
-     Collects device data for PayPal.
+     Collects device data.
 
-     This should be used when the user is paying with PayPal or Venmo only.
-     - Returns: A deviceData string that should be passed into server-side calls, such as `Transaction.sale`, for PayPal transactions. This JSON serialized string contains a PayPal fraud ID.
+     - Returns: A device data string that should be passed into server-side calls, such as `Transaction.sale`. This JSON serialized string contains a PayPal fraud ID.
```
Comment (scannillo) [inline Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift:25]: `PayPal fraud ID` is this language we use in the docs? I don't think we use this anywhere else. 

Maybe we stick with `correlationID`? Writing this because it could confuse a merchant to think this ID is only applicable for PayPal checkouts.

Comment (sestevens) [reply Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift:25]: I didn't actually see any specific name for this in our docs. Maybe we could just reword this to something like:

```suggestion
     - Returns: A JSON serialized string containing device data that should be passed into server-side calls, such as `Transaction.sale`.
```

Comment (sestevens) [reply Sources/PayPalDataCollector/Public/PayPalDataCollector/PPDataCollector.swift:25]: Or maybe instead of "containing device data" it should be "containing a device data identifier", since it doesn't actually contain raw device data.

**Review by sshropshire — APPROVED** [review]

**Review by sarahkoop — APPROVED** [review]
